src/test/java/hudson/remoting/ClassRemotingTest.java

```diff
@@ -50,6 +51,7 @@ public class ClassRemotingTest {
     static final String TESTCALLABLE_TRANSFORMED_CLASSNAME = "hudson.rem0ting.TestCallable";
     static final String TESTLINKAGE_TRANSFORMED_CLASSNAME = "hudson.rem0ting.TestLinkage";
 
+    @Disabled("TODO flakes: Artificial testing interrupt.")
```

Comment:
Not sure what is going on with these. Interference from https://github.com/jenkinsci/remoting/blob/b5eba8204c1e200a6a225e806e80c86e36f45710/src/test/java/hudson/remoting/ClassRemoting2Test.java#L325 (in a different source!) suggests something structurally wrong with the tests.